browser/src/ConstraintTrack.tsx

```diff
@@ -52,6 +53,36 @@ type Props<R extends GenericRegion> = {
   valueFn: (region: R) => string
 }
 
+export const regionsInExons = <R extends GenericRegion>(regions: R[], exons: Exon[]): R[] => {
+  const sortedRegions = regions.sort((a, b) => a.start - b.start)
+  const sortedExons = exons.sort((a, b) => a.start - b.start)
+
```

Comment:
Woohoo, very nicely done.

I honestly avoided touching that function after extending the tests for it since it was a bit unreadable to my eyes at that time, this goes a long way towards making it approachable -- kudos.

Reply:
refactoring this bit was very satisfying